### PR TITLE
Feature/partial listing

### PR DESCRIPTION
**Pull request by swey**
This PR is an addition to https://github.com/unic/estatico/pull/54

Additionally to linking the used sub modules, this change will also list all used partials (which are no own module). Looks like this:
bildschirmfoto 2017-03-09 um 16 47 27

### CODE REVIEW

**Review by orioltf — APPROVED** [review]

File: helpers/data.js [hunk 1]
```diff
@@ -71,12 +71,59 @@ module.exports = {
 	},
 
 	getTemplateCode: function(filePath) {
+		function highlightTemplateCode(content) {
```
Comment (orioltf) [inline helpers/data.js]: Can we have this function in the main scope (like `getTemplateCode` or `getUsedPartialsInTemplate`)?

**Review by orioltf — CHANGES_REQUESTED** [review]

**Comment by swey** [comment]
@orioltf Please re-check (Also made those new methods `private`)

**Comment by orioltf** [comment]
Hi @swey would you mind resolving the conflicts, please? Thanks!

**Comment by swey** [comment]
Next try :)

**Review by orioltf — APPROVED** [review]

**Pull request merged**